migration/migrating_3_4/troubleshooting-3-4.adoc

```diff
@@ -6,7 +6,7 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can view the migration Custom Resources (CRs) and download logs to troubleshoot a failed migration.
+You can view the {mtc-short} custom resources and download logs to troubleshoot a failed migration.
```

Comment:
@apinnick thank you for your updates. Regarding my comment here, this was only in relation to the first time you reference the Migration Toolkit for Containers in an assembly or in a module. In your changes it looks like you have applied `{mtc-full}` throughout most of the PR. My suggestion was only to update `{mtc-short}` with `{mtc-full}` when first mentioning the product in a module or in an assembly. This way, if a module is reused, the reader will be presented with the full product name and then subsequently the acronym.

Can you please update this again throughout the PR to reflect this? Please let me know if you have any questions.

Reply:
> In your changes it looks like you have applied `{mtc-full}` throughout most of the PR. 

I don't think so. I will check.

Reply:
On review I noticed that the change from `{mtc-short}` to `{mtc-full}` that I referred to were specifically in relation to the operator name. This looks good to me.

Reply:
That was because I saw that the actual name of the Operator is "Migration Toolkit for Containers Operator". At one point, the name was going to be "MTC Operator".